src/Core/Components/AppBar/IAppBarItem.cs

```diff
@@ -0,0 +1,67 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Routing;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+public interface IAppBarItem
+{
+    /// <summary>
+    /// Gets or sets the unique identifier for this item.
+    /// If implementing this interface in a FluentComponentBase derived component, it will be implemented already.
+    /// </summary>
+    public string? Id { get; set; }
+
+    /// <summary>
+    /// Gets or sets the URL for this item.
+    /// </summary>
+    public string? Href { get; set; }
+
+    /// <summary>
+    /// Gets or sets how the link should be matched.
+    /// Defaults to <see cref="NavLinkMatch.Prefix"/>.
+    /// </summary>
+    public NavLinkMatch Match { get; set; }
```

Comment:
This property is not used in the code?

Reply:
Yes, it is being used:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/a720bd99-8eef-4d45-a7b5-969a15982d04)

Reply:
Hups... sorry :-)